AtReconstruction/AtFilterTask.cxx

```diff
@@ -71,7 +71,8 @@ void AtFilterTask::Exec(Option_t *opt)
    if (fFilterFPN)
       for (auto &[ref, pad] : filteredEvent->fFpnMap) {
          LOG(debug) << "Filtering " << ref;
-         fFilter->Filter(&pad);
+         AtPadReference padRef = ref;
```

Comment:
Is this line necessary? I think ref should already be of the type AtPadReference &ref